### ISSUE TITLE
Input and output spatial dimensions differ

### ISSUE THREAD

**Issue by Spiruel** [sentinel-hub/eo-flow#35]
Hi,

When reading through the example notebook I see that the input dimensions are of shape eg (20, 128, 128, 10) yet the output predictions are (20, 60, 60, 10). I am wondering whether there is an explanation as to why the spatial dimensions are different and whether you would be able to clarify this?

Perhaps you on have on hand a full print of the `model.summary()` so one can see what is occurring at each layer?

Cheers

**Comment by devisperessutti** [comment]
Hi @Spiruel 

This is due to using only the `VALID` part of the convolutions, e.g. the outer pixel of the input features is not used, and the output features have a smaller size of 2 pixels per dimension for each convolutional block.

If you don't want this behaviour, simply change the padding to `SAME` in the config file and the output will have the same size as the input image. More info about padding [here](https://keras.io/api/layers/convolution_layers/convolution2d/).

**Comment by Spiruel** [comment]
That makes sense - thanks for the clarification!
